include/quda.h

```diff
@@ -799,8 +799,19 @@ extern "C" {
                       const void* const u_link,
                       const QudaGaugeParam* param);
 
+  /**
+   * Computes the total, spatial and temporal plaquette averages of the loaded gauge configuration.
+   * @param Array for storing the averages (total, spatial, temporal)
+   */
+  void plaqQuda(double plaq[3]);
```

Comment:
Well, that could be easily done. Actually, if you want, we can call the two functions the same (plaqCuda I assume) and overload them, since they have different arguments.

> El 7/7/2015, a las 22:28, Mathias Wagner <notifications@github.com> escribió:
> 
> In include/quda.h <https://github.com/lattice/quda/pull/307#discussion_r34086173>:
> 
> > @@ -799,8 +799,19 @@ extern "C" {
> >                        const void* const u_link,
> >                        const QudaGaugeParam* param);
> >  
> > +  /**
> > +   * Computes the total, spatial and temporal plaquette averages of the loaded gauge configuration.
> > +   * @param Array for storing the averages (total, spatial, temporal)
> > +   */
> > +  void plaqQuda(double plaq[3]);
> As we want to merge that into the 0.7.x release and this changes the interface: Shall we just for the sake of avoid to break anything also keep the double plaqCuda() signature (which should be plaqQuda).
> 
> This would only be relevant if this is used and would break a code.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/lattice/quda/pull/307/files#r34086173>.
>